### ISSUE TITLE
docs: Detail `talosctl disks` usage in maintentance mode

### ISSUE THREAD

**Issue by rsmitty** [siderolabs/talos#4510]
This is follow-up from https://github.com/talos-systems/talos/issues/4507, but we should explicitly mention how to obtain the disk in maintenance mode.

**Comment by sergelogvinov** [comment]
as an idea. Inventory api call.

one method to get most of popular server parameters like:
* cpu
* memory
* disks
* lspci
* lsusb
* property from ipmi/ilo?

**Comment by smira** [comment]
yep, that would be cool to do more stuff like that, we can return SMBIOS info, etc.
